Library/ViewModels/PledgeViewModel.swift

```diff
@@ -1215,7 +1215,7 @@ private func checkoutPropertiesData(
 
 /* A helper that assists in rounding the dollar converted amount to a given number of decimal places
  */
-private func rounded(_ value: Double, places: Int) -> Double {
+internal func rounded(_ value: Double, places: Int) -> Double {
```

Comment:
Instead of making this internal to this view model you should move it to `SharedFunctions` in the Library and ensure it's covered by tests.

Reply:
Okay, noted.